src/waltz/http/fd_http_server.c

```diff
@@ -464,7 +466,7 @@ read_conn_http( fd_http_server_t * http,
 
   ulong content_len = 0UL;
   ulong content_length_len = 0UL;
-  if( FD_UNLIKELY( method_enum==FD_HTTP_SERVER_METHOD_POST ) ) {
+  if( FD_UNLIKELY( method_enum==FD_HTTP_SERVER_METHOD_POST || method_enum == FD_HTTP_SERVER_METHOD_PUT ) ) {
```

Comment:
match existing style please

```suggestion
  if( FD_UNLIKELY( method_enum==FD_HTTP_SERVER_METHOD_POST || method_enum==FD_HTTP_SERVER_METHOD_PUT ) ) {
```